build/packages-template/bbb-web/build.sh

```diff
@@ -84,6 +84,9 @@ cp ../bbb-web.env "$STAGING"/etc/default/bbb-web
 mkdir -p "$STAGING"/lib/systemd/system
 cp ../bbb-web.service "$STAGING"/lib/systemd/system
 
+cp ../bbb-process-image.sh "$STAGING"/usr/bin
+cp ../bbb-process-pdf-page.sh "$STAGING"/usr/bin
```

Comment:
You are installing the scripts to /usr/bin but bbb-web is hardcoded to run them from /usr/local/bin